_data/projects/wospm-checker.yml

```diff
@@ -0,0 +1,12 @@
+name: WOSPM Checker
+desc: A checker for open source project owners to measure their open source project if welcoming or not
```

Comment:
@umutphp Can we improve the Description? last few lines are not properly understood

Reply:
I have changed the description a bit. Can you please check it again?